test/_helpers.js

```diff
@@ -1,8 +1,23 @@
 import 'babel-register';
```

Comment:
I missed that in the previous :grin: `babel-register` and `babel-polyfill` should be imported only at entry points (it's not an error, just a redundancy)